src/main/kotlin/frc/robot/subsystems/gripper/GripperIOReal.kt

```diff
@@ -28,7 +32,7 @@ class GripperIOReal : GripperIO {
     }
 
     override fun setPower(power: Double) {
-        motor.set(power)
+        motor.setControl(control.withOutput(power * RobotController.getBatteryVoltage()))
```

Comment:
I don't think that's how it's done. What you did is basically duty out. Just pass it a voltage.
```suggestion
        motor.setControl(control.withOutput(power))
```
Where `power` is voltage (e.g. 5V to outtake and (-5V) to intake.